### ISSUE TITLE
Plugin not working on SublimeText 3 ?

### ISSUE THREAD

**Issue by p-j** [spadgos/sublime-jsdocs#236]
My configuration is as follow : 
- OS: Linux (Kubuntu 12.10)
- ST3 Build 3047
- Latest version of DocBlockr from [Package Control](https://sublime.wbond.net/packages/DocBlockr)

Now, when I write Javascript or PHP, if I type `/**` and hit enter or tab, it just insert a new line or a tab.

Same with `/*` or `///`.

Here are the packages I have installed (maybe it's a conflicting problem that someone knows of).

```
AlignTab
CodeFormatter
DocBlockr
EditorConfig
FileDiffs
GitGutter
Handlebars
MarkdownEditing
Markdown Extended
Markdown Preview
Monokai Extended
Open URL
Package Syncing
SublimeGit
Syntax Highlighting for Sass
Syntax Manager
```

I just updated all of them in case it was a known and already fixed issue, but no luck.

Any idea why it's not working ?

**Comment by Bazm0** [comment]
solution proposed by @alexandrupirjol  works for me on Mac.

**Comment by empireryan** [comment]
@alexandrupirjol 's solution does not work for me on OS X

**Comment by izevg** [comment]
Yep, solved by removing all User Key Bindings and restarting Sublime

**Comment by ali-khabbaz** [comment]
reinstall worked for me !
sublime text 3
win 8.1

**Comment by rickmed** [comment]
none of the above solutions worked for me. ST3083, win10x64

**Comment by oradwell** [comment]
Mine stopped working when I updated to ST3099. Fedora 23. Reinstalling didn't fix the issue.

**Comment by spadgos** [comment]
@oradwell could you post your console output (instructions in a comment above https://github.com/spadgos/sublime-jsdocs/issues/236#issuecomment-33314717)

**Comment by oradwell** [comment]
@spadgos There doesn't seem to be any errors and I was able to see `command: insert {"characters": "\n"}` in console. I don't have a user keymap file.

**Comment by spadgos** [comment]
That would suggest that the DocBlockr plugin isn't getting invoked. :-\

**Comment by artdevgame** [comment]
[Regression fixed in SL 3102](https://github.com/spadgos/sublime-jsdocs/issues/406#issuecomment-181296392)

**Comment by oradwell** [comment]
Thanks @flyingbuddha 3102 update fixed the issue.

**Comment by harri00413** [comment]
Hugofy not working for me either on OSX. Why put it up there if it isn't working? I'll go back to cli hugo. Too bad.

**Comment by ceciliassis** [comment]
@alexandrupirjol solution, didn't work for me.

But a fresh install of ST and plugins, did.
